application_form/services/offer.py

```diff
@@ -87,3 +94,16 @@ def update_reservations_based_on_offer_expiration(
         reservation.set_state(ApartmentReservationState.OFFERED, user=user)
 
     return new_expired_reservations.count(), not_anymore_expired_reservations.count()
+
+
+def update_other_reservation_state_in_queue(reservation):
+    apartment = get_apartment(reservation.apartment_uuid)
+    other_reservations = ApartmentReservation.objects.filter(
+        apartment_uuid=reservation.apartment_uuid
+    ).exclude(Q(state=ApartmentReservationState.CANCELED) | Q(id=reservation.id))
+    for reservation in other_reservations:
+        cancel_reservation(
+            reservation,
+            cancellation_reason=ApartmentReservationCancellationReason.APARTMENT_OFFERED.value,  # noqa: E501
```

Comment:
Hmm, I don't think this should need `.value` 🤔

Reply:
yeah, you are correct, fortunately that it still works :D Fixed it anyway